src/data-source.handlers.ts

```diff
@@ -24,7 +23,7 @@ export const dataSourceHandlers: ResourceHandlerMap = {
       .save()
       .then(datasource => {
         res
-          .location(`${dataSourcesPath}/${datasource.id}`)
+          .location(datasource.id)
```

Comment:
Although the URI is not exactly meaningful or unique that way, it's easier to parse on the FE

Reply:
Oh actually I think it's necessary with a / in the beginning if it's relative. The idea is to be able to do originalUrl + location. Original url here being `{host}/api/datasources` and so location should be `'/${datasource.id}'` and not just `datasource.id`

Reply:
jeg tror det ikke. / mener absolutt fra host.

Reply:
ok!